### PR TITLE
Add Github Actions

### PR DESCRIPTION
**Pull request by DonRenando**
#3 Add Github Actions to test and build go code

### CODE REVIEW

File: .github/workflows/go.yaml [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+    runs-on: ${{ matrix.os }}
+    steps:
+    - name: Install Go
+      uses: actions/setup-go@v2
+      with:
+        go-version: ${{ matrix.go-version }}
+    - name: Checkout code
+      uses: actions/checkout@v2
+    - name: Test
+      run: go test -v ./...
```
Comment (jmaitrehenry) [inline .github/workflows/go.yaml:18]: Can you add `--cover` option?

**Review by jmaitrehenry — COMMENTED** [review]
Thanks for your PR, it's really clean!
Can you just add the cover option and I will merge it?

File: .github/workflows/go.yaml (hunk 1 above)
Comment (DonRenando) [reply .github/workflows/go.yaml:18]: yes of course :)

**Review by jmaitrehenry — APPROVED** [review]
LGTM, thanks for your contribution!

**Pull request merged**